modules/DrugSynonymDataToNeo4j.py

```diff
@@ -14,7 +15,7 @@ def property_type_checker(property_value):
         elif isinstance(property_value,str):
             property_value = '''"''' + property_value.replace('"',r"\"") + '''"'''
         elif not property_value:
-            property_value = ""
+            property_value = "''"
```

Comment:
@bechbd  not sure of best practices here wrt neo4j

Reply:
this is only if the property value is None